packages/web-components/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@microsoft/fast-components-msft",
+  "description": "A library of Fluent Web Components",
+  "sideEffects": false,
+  "version": "1.6.0",
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/fast/issues/new/choose"
+  },
+  "main": "dist/esm/index.js",
+  "types": "dist/fast-components-msft.d.ts",
+  "unpkg": "dist/fast-components-msft.min.js",
+  "scripts": {
```

Comment:
For this package to build properly with the rest of the repo, there may be some modifications needed to the scripts and/or to the CI job. 

We use `lerna run` to run the following scripts in each package:
- `clean`
- `build`: core build stuff, usually running TS but not bundling (though this could vary by package)
- `bundle`: runs webpack if appropriate (not all packages have this)
- `lint`: runs eslint
- `prettier`: runs prettier (*without* `--write` option by default)
- `test`: runs tests (mostly unit tests)

If there are any other scripts that you need run as part of the PR/CI job, `azure-pipelines.yml` will need to be modified to handle that.

Reply:
Updated the task to `clean` from `clean:dist` and `eslint` to `lint`. I've dropped prettier as it doesn't seem to be defined for any of the other packages and running at the root accomplishes what's needed. While we have several tests denoted in here, the primary task is labelled `test`. Hopefully that reconciles where need be!